src/Hydration.js

```diff
@@ -1,23 +1,32 @@
+import {averager,  findDataByDate,  finder}  from './util';
+
 class Hydration {
   constructor(hydrationData) {
     this.hydrationData = hydrationData;
   }
+
   calculateAverageOunces(id) {
-    let perDayUserHydration = this.hydrationData.filter((data) => id === data.userID);
-    return perDayUserHydration.reduce((sumSoFar, data) => {
-      return sumSoFar += data.numOunces;
-    }, 0) / perDayUserHydration.length;
+    let perDayUserHydration = this.hydrationData.filter(data => id === data.userID);
+    return averager(perDayUserHydration, `numOunces`);
   }
+
   calculateDailyOunces(id, date) {
-    let findOuncesByDate = this.hydrationData.find((data) => id === data.userID && date === data.date);
-    return findOuncesByDate.numOunces;
+    //let findOuncesByDate = this.hydrationData.find((data) => id === data.userID && date === data.date);
+    //console.log(finder(this.hydrationData, id, date).numOunces)
+    return finder(this.hydrationData, id, date).numOunces;
+    //return findOuncesByDate.numOunces;
   }
```

Comment:
I appreciate you telling me in slack that these comments are only to be used as reference to the old code